crates/sui-graphql-rpc/src/types/epoch.rs

```diff
@@ -66,9 +66,10 @@ impl Epoch {
             .await?;
 
         let active_validators = convert_to_validators(
-            system_state.active_validators,
-            None,
+            system_state.clone().active_validators,
+            Some(system_state.clone()),
```

Comment:
this seems strange, wouldnt we always have system_state? can we reduce the function params to just taking in non-optional system_state?

Reply:
Let me revisit this, thanks for noticing!